File: app/src/main/java/com/dominikgaller/alpinebooking/booking/core/inport/result/ConfirmTourBookingResult.java
package com.dominikgaller.alpinebooking.booking.core.inport.result;

/**
 * Result returned by the ConfirmTourBooking use case.
 *
 * <p>Contains only standard-library types — no domain value objects.
 *
 * <p>SDD: See {@code documentation/use-cases/uc02-confirm-tour-booking.spec.md}, section 3.
 */
public record ConfirmTourBookingResult(String status) {
}

File: app/src/main/kotlin/com/example/service/booking/core/inport/result/ConfirmTourBookingResult.kt
package com.example.service.booking.core.inport.result

/**
 * Output of the ConfirmTourBooking use case (UC02).
 *
 * SDD: see `documentation/use-cases/uc02-confirm-tour-booking.spec.md` § 3.
 */
data class ConfirmTourBookingResult(
    val status: String,
)
